src/act/random.rs

```diff
@@ -36,5 +40,21 @@ pub fn generate_randomness_implementation() -> TokenStream {
                 });
             });
         }
+
+        #[cfg(all(target_arch = "wasm32", target_os = "wasi"))]
+        fn rng_seed() {
+            ic_cdk::spawn(async move {
+                let result: ic_cdk::api::call::CallResult<(Vec<u8>,)> = ic_cdk::api::call::call(
+                    candid::Principal::from_text("aaaaa-aa").unwrap(),
```

Comment:
😱

Reply:
We don't have runtime error infrastructure for the cdk_framework yet, so I'm punting this to whoever does that

Reply:
I've removed two unwraps by using the management_canister API directly, the rest were already there and we haven't done a pass to figure out runtime errors in the cdk_framework yet